### DISCUSSION TITLE
✨ Get featured on Shime

### DISCUSSION THREAD

**Discussion by ryanlua** [ryanlua/Shime#6] [Announcements]
> Submit your work and get featured on Shime.

Shime to looking to see what players have created using Shime. Post your work [Showcase](https://github.com/RyanLua/Shime/discussions/categories/showcase) category and get entered in.

If you are featured you will be accredited with your name, link, and media of your work. You are not needed to release your work at all, only proof you own the work, and a picture or video of the work is good enough. We will not claim ownership of your work or licensing along with similar rights.

Featured works will be displayed throughout Shime material such as the README, Wiki, DevForum Post, and more in the future. 

We can't wait to see what you all have created using Shime!

**Comment by ghost** [comment]
I used the module to show that there is nothing in this game

https://www.roblox.com/games/10567027426/Clean-Up-VSC-Team

**Reply by ryanlua** [reply]
> I used the module to show that there is nothing in this game
> 
> https://www.roblox.com/games/10567027426/Clean-Up-VSC-Team

It seems like the experience is private. You can post media in the [showcase category](https://github.com/RyanLua/Shime/discussions/categories/showcase) if you prefer your experience remain private.

**Reply by ghost** [reply]
Just made it public (to let you know)

**Comment by ryanlua** [comment]
I've decided that anyone can be featured on Shime. This will be kept open indefinitely.
